core/conversion/converters/impl/interpolate.cpp

```diff
@@ -0,0 +1,113 @@
+#include "torch/torch.h"
+#include "core/util/prelude.h"
+#include "core/conversion/converters/converters.h"
+
+#include <csignal>
+
+namespace trtorch {
+namespace core {
+namespace conversion {
+namespace converters {
+namespace impl {
+namespace {
+
+auto interpolate_registrations = RegisterNodeConversionPatterns()

```

Comment:
Can you add the `TRTORCH_UNUSED` label here to suppress warnings on other platforms?